srcpkgs/rock64-uboot/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'rock64-uboot'
+pkgname=rock64-uboot
+version=2020.07
+revision=1
+archs="aarch64*"
+wrksrc="u-boot-${version}"
+build_style=gnu-makefile
```

Comment:
that's obsolete if you provide all of the `do_*()` functions, no?

Reply:
yep thanks for pointing that out, removed the line